etc/modules-load.d/zfs.conf.in

```diff
@@ -1,3 +1,2 @@
-# Always load kernel modules at boot.  The default behavior is to load the
-# kernel modules in the zfs-import-*.service or when blkid(8) detects a pool.
-#zfs
+# Always load kernel modules at boot.
+zfs
```

Comment:
In fact, this used to be enabled by default, see 92547bc45ca9a2114662d9343ae53e5098acb627.  However, we ended up disabling the default since we had multiple complaints about the modules always being loaded even though ZFS wasn't being used on the system. 

What if instead of re-enabling this by default we did a `touch /etc/zfs/zpool.cache` as part of the RPM install before starting the services.  As far as ZFS is concerned an empty cache file is equivilant to a missing cache file so there's no functionalty change.  However, the existance of the empty file will trigger the module load.

The advtange of this would be that the automatic module loading behavior would be more discoverable.  It turns out that the `modules-load.d` directory isn't  well known functionality yet and users found it difficult to determine _why_ the modules were loading.